### PR TITLE
Docs: Adding development process and coding style sections

### LINKED ISSUE #12: Add code style guides to the 'Contributing' section of our docs
We should give better instructions for developers who are new to project to get started. Including our various style guides, how to go about claiming an issue, how to request feedback, etc.

### PR DESCRIPTION
**Pull request by tonyjin**
### Description:

Resolves #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,11 @@ Contributions are welcome!
 
 Clone locally with:
 
-    git clone https://github.com/OriginProtocol/docs.git origin-docs && origin-docs
+    git clone https://github.com/OriginProtocol/docs.git origin-docs && cd origin-docs
+
+Make sure you have Ruby installed or install (e.g. with Homebrew on Mac):
+
+    brew install ruby
```
Comment (micahalcorn) [inline README.md]: Need ruby > 2.3.1 and I would generally recommend `rvm`, but people may be fine with `brew`

File: source/includes/_contributing.mdown [hunk 2]
```diff
@@ -1,32 +1,41 @@
...
+# Contributing
 
-## Discord
+Want to hack on Origin? Awesome!
 
-Our team collaboration is done in public and our company Discord is open to all. If you have questions or need help getting started, our [Discord #engineering channel](https://www.originprotocol.com/discord) is a great place to get assistance from our team of engineers and developers.
+Origin is an Open Source project and we welcome contributions of all sorts. There are many ways to help, from reporting issues, contributing code, and helping us improve our community.
 
-## Email
+There are four main repositories:
```
Comment (micahalcorn) [inline source/includes/_contributing.mdown]: Many of the "good first issues" are actually on the company-website repo, or even this one. We may or may not want to steer people toward the product-related repos (the four mentioned here). For that matter, "good first issues" are generally fairly simple and _some_ of our target audience (blockchain and advanced developers) might be more interested in tackling a more "interesting" issue. Maybe we suggest that people search the issues by programming language or another label best done [here](https://github.com/OriginProtocol).

File: source/includes/_contributing.mdown [hunk 3]
```diff
@@ -1,32 +1,41 @@
...
+- [Origin.js](https://github.com/OriginProtocol/origin-js) - solidity contracts and the Origin JS API
+- [Demo DApp](https://github.com/OriginProtocol/demo-dapp) - example react powered front end 
+- [Bridge Server](https://github.com/OriginProtocol/bridge-server) - python indexing and search server
+- [Identity Playground](https://github.com/OriginProtocol/identity-playground) - implementation of ERC 725/735 for blockchain identity
 
-# Contributing
+### Dive Right In
 
-Want to hack on Origin? Awesome!
+If you're ready to start hacking on Origin right now and you just need an issue to focus on, check out [this list of good first issues](https://github.com/orgs/OriginProtocol/projects/2?card_filter_query=label%3A%22good+first+issue%22+no%3Aassignee).
```
Comment (micahalcorn) [inline source/includes/_contributing.mdown]: Only the four product-related repos are guaranteed to have their issues in the project board. So many of the "good first issues" in the docs repo and company-website repo will be found by searching [here](https://github.com/OriginProtocol).

File: source/includes/_contributing.mdown (hunk 2 above)
Comment (joshfraser) [reply source/includes/_contributing.mdown]: This documentation is really focused on the 3 platform-related repos. 

We should probably separate the identity playground, but let's mention the other repos like the company website as other places that are sometimes better to get started.

File: source/includes/_contributing.mdown (hunk 3 above)
Comment (micahalcorn) [reply source/includes/_contributing.mdown]: Actually, the search experience at that top level of the GitHub organization is not very intuitive. This is another of several reasons to add the company-website repo to the project board.

**Comment by joshfraser** [comment]
Missed this on my first pass, but a few of the relative links are broken. For example, we link to 

http://docs.originprotocol.com/#code-style

but the path should actually be:

http://docs.originprotocol.com/#contributing-email-coding-style

Same goes for #community-guidelines and #development-process

**Comment by tonyjin** [comment]
Hmm something's wrong with the headers, some are using ## and some are using ###. I'll resolve.

**Comment by wanderingstan** [comment]
@joshfraser, the broken relative links are almost certainly because of #15 , where we merged an upstream fix from lord/slate with anchor-linking to headings that weren't unique. Makes sense that this would break existing links and we should have checked for that.

**Pull request merged**